modules/end-user-guide/pages/enabling-maven-artifact-repositories.adoc

```diff
@@ -0,0 +1,108 @@
+:navtitle: Maven
+:keywords: maven, artifact-repository, artifact-repositories
+:page-aliases: .:using-maven-artifact-repositories
+
+[id="enabling-maven-artifact-repositories"]
+= Enabling Maven artifact repositories
+
+To enable a Maven artifact repository in Maven workspaces that run in a restricted environment:
+
+.Prerequisites
+
+* You are not running any Maven workspace.
+
+.Procedure
+
+. Apply the TLS certificate as a Secret:
++
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: Secret
+apiVersion: v1
+metadata:
+  name: tls-cer
+  annotations:
+    controller.devfile.io/mount-path: /home/user/certs
+    controller.devfile.io/mount-as: file
+  labels:
+    controller.devfile.io/mount-to-devworkspace: 'true'
+    controller.devfile.io/watch-secret: 'true'
+data:
+  tls.cer: >-
+    __<Base64_encoded_content_of_public_cert>__ <1>
+----
+<1> Base64 encoding with disabled line wrapping.
+
+. Apply the ConfigMap to create the `settings.xml` file:
```

Comment:
You have 6 examples with "Apply the ConfigMap to ..." and 4 examples with "Apply ... as a ConfigMap."

Reply:
I believe this is resolved now.